tests/v2/codecoverage/package/Dockerfile

```diff
@@ -156,13 +158,13 @@ RUN ln -s /usr/bin/cni /usr/bin/bridge && \
 
 RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     mkdir -p /var/lib/rancher/k3s/agent/images/ && \
-    curl -sfL ${ETCD_URL} | tar xvzf - --strip-components=1 -C /usr/bin/ etcd-${CATTLE_ETCD_VERSION}-linux-${ARCH}/etcdctl && \
+    curl -sfL ${ETCD_URL} | tar xvzf - --strip-components=1 --no-same-owner -C /usr/bin/ etcd-${CATTLE_ETCD_VERSION}-linux-${ARCH}/etcdctl && \
     curl -sLf https://github.com/rancher/telemetry/releases/download/${TELEMETRY_VERSION}/telemetry-${ARCH} > /usr/bin/telemetry && \
     chmod +x /usr/bin/tini /usr/bin/telemetry && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.7.2-rc8
-ENV CATTLE_DASHBOARD_UI_VERSION v2.7.2-rc8
+ENV CATTLE_UI_VERSION 2.7.5-rc2
```

Comment:
Should we update this to 2.7.5-rc3 as that's the final RC?

Reply:
Well that's what in the release/v2.7 main branch. I update it according to that docker file.

Reply:
Maybe should another update once we release v2.7.5

Reply:
Whichever makes more sense for the code coverage, that just caught my eye during a review. Everything else lgtm

Reply:
Do we want to update this as and when a new rancher version releases? not that it is essential but thought would ask.

Reply:
Yes I was planning on updating it after the new rancher releases. What do you all think?

Reply:
sgtm, we had a similar process for the old ontag jobs